lib/allure_report_publisher/lib/providers/gitlab.rb

```diff
@@ -58,28 +58,67 @@ def update_pr_description
         )
       end
 
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Yeah, I think it's fine. I might take a look if the class can be refactored at some point in future if I have nothing better to do 😄 

```suggestion
      # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```